keyboards/namecard2x4/keymaps/default/keymap.c

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2018 REPLACE_WITH_YOUR_NAME
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum Layer
+{
+	DF=0,
```

Comment:
You don't need to specify 0 here, It starts at that, and increments automatically. 

Also, could you format it like this: 
```c
enum Layer {
	DF,
	LW,
	RS
};
```

Reply:
I remove explicit initialization.